test/js/ez-navigation-hub.js

```diff
@@ -7,4 +12,128 @@ describe('ez-navigation-hub', function() {
             element.constructor
         );
     });
+    describe('properties', function () {
+        describe('`activeZoneClass`', function () {
+            it('should have a default value', function () {
+                assert.equal(element.activeZoneClass, element.getAttribute('active-zone-class'));
+            });
+
+            it('should set', function () {
+                const value = 'Something';
+
+                element.activeZoneClass = value;
+                assert.equal(element.activeZoneClass, value);
+            });
+        });
+
+        describe('`matchedLinkClass`', function () {
+            it('should have a default value', function () {
+                assert.equal(element.matchedLinkClass, element.getAttribute('matched-link-class'));
+            });
+
+            it('should set', function () {
+                const value = 'Something';
+
+                element.matchedLinkClass = value;
+                assert.equal(element.matchedLinkClass, value);
+            });
+        });
+
+        describe('`activeZone`', function () {
+            it('should default to undefined', function () {
+                assert.isUndefined(element.activeZone);
+            });
+
+            it('should set', function () {
+                const value = 'contents';
+
+                element.activeZone = value;
+                assert.equal(element.activeZone, value);
+            });
+        });
+
+        describe('`matchedLinkUrl`', function () {
+            it('should default to undefined', function () {
+                assert.isUndefined(element.matchedLinkUrl);
+            });
+
+            it('should set', function () {
+                const value = 'url1';
+
+                element.matchedLinkUrl = value;
+                assert.equal(element.matchedLinkUrl, value);
+            });
+        });
+    });
+    describe('active zone update', function () {
+        function simulateClick(element) {
+            const click = new CustomEvent('click', {
+                bubbles: true,
+                cancelable: true,
+            });
+
+            element.dispatchEvent(click);
+            return click;
+        }
+        function assertEventHandled(event) {
+            assert.ok(
+                event.defaultPrevented,
+                'The event should have been prevented'
+            );
+        }
+        describe('highlight zone', function () {
+            it('should highlight zone contents', function () {
```

Comment:
bad name, the `contents` is part of your fixtures, this has nothing to do what the navigation hub is doing, it's an example for your test. Should be named with something like `should highlight the selected zone`